media_daemon/auto_skills.py

```diff
@@ -27,10 +27,10 @@ def make_skill(): # pylint-enable
     for skill in skills:
         f.write(skill + '\n')
     f.close()
-    shutil.move( media_daemon_folder + 'custom_skill.txt', server_skill_folder)
-    f2 = open(server_settings_folder + 'customized_config.properties','a')
-    f2.write('local.mode = true')
-    f2.close()
+    shutil.move(os.path.join(media_daemon_folder, 'custom_skill.txt'), server_skill_folder)
+    with open(os.path.join(server_settings_folder, 'customized_config.properties'), 'a') as f2:
+        f2.write('local.mode = true')
+        f2.close()
```

Comment:
Oops, you didn't read documentation about `with open()`?
With the use of that feature, no need to call `f2.close()`.